readablepdf

```diff
@@ -354,11 +358,22 @@ while true; do
 	esac
 done
 
+
+echo "Estimating CPU-Threads"
+# Estimate CPU Cores (see pdfsandwich code for reference)
+CPU_THREADS=$(cat /proc/cpuinfo | grep processor | awk '{a++} END {print a}')
+OMP_THREAD_LIMIT=$CPU_THREADS
+MAX_JOBS=$(expr $CPU_THREADS / 4)
+# tesseract only uses 4 threads
+
+echo "Using up to $CPU_THREADS threads."
+
 # Use GNU Parallel to speed things up if it exists.
+
 if command -v parallel >/dev/null; then
-	parallel tesseract "{}" "{.}" "${TESSERACT_PARAMS}" hocr ::: *.tif
+	parallel -P ${MAX_JOBS} tesseract "{}" "{.}" "${TESSERACT_PARAMS}" hocr ::: *.tif
 else
-	for i in ./*.tif; do tesseract "$i" "$(basename "$i")" "${TESSERACT_PARAMS}" hocr; done;
+	for i in ./*.tif; do tesseract $i $(basename "$i") ${TESSERACT_PARAMS} hocr; done;
```

Comment:
Wouldn't this change introduce highly undesirable word splitting?

Reply:
I guess it is more secure to use quotes, so yes.